### ISSUE TITLE
The result returned is un-named

### ISSUE THREAD

**Issue by drop19** [Destiner/ethcall#114]
For example, when calling `Pair.getReserves()` in uniswap.
The typechain result is
```
getReserves(
    overrides?: CallOverrides
  ): Promise<
    [BigNumber, BigNumber, number] & {
      reserve0: BigNumber;
      reserve1: BigNumber;
      blockTimestampLast: number;
    }
  >;
```

However the returned object form this library is just `[BigNumber, BigNumber, number]`, the named object is missing

**Comment by Destiner** [comment]
Hey @drop19, thanks for feedback.

Should be fixed in `4.7.2`. Please let me know if the issue persists.

**Comment by drop19** [comment]
I appreciate your help!
